packages/api-docs/src/types.ts

```diff
@@ -50,7 +50,7 @@ export interface ITableJson {
    * `[{ text: 'Readonly<' }, { text: 'IFoo', linkedPage: 'Foo', linkedPageGroup: 'components' }, { text: '>' }]`
    */
   extendsTokens?: ILinkToken[];
-  members?: ITableRowJson[] | IEnumTableRowJson[];
+  members: ITableRowJson[] | IEnumTableRowJson[];
```

Comment:
Just want to confirm, we're making the `members` required because there is no reason to have a table without them right?

Reply:
The main goal was to address the mismatch between this interface and the mirrored one in DocPage.types.ts. Thinking about it more, I'm going to revert this and make members optional in the other interface (and update the spot in ApiReferencesTableSet which assumed members was defined), because type aliases don't have members.